query/src/main/scala/filodb/query/util/HierarchicalQueryExperience.scala

```diff
@@ -157,20 +169,27 @@ object HierarchicalQueryExperience extends StrictLogging {
    *  metricSuffix = agg_2
    *  Existing metric name - metric1:::agg
    *  After update - metric1:::agg -> metric1:::agg_2
+   * @param metricName - String - Metric ColumnFilter tag/label
+   * @param metricRegex - HierarchicalQueryExperience - Contains
+   * @param metricSuffix - Seq[ColumnFilter] - label filters of the query/lp
+   * @return - Option[String] - Next level aggregated metric name
+   */
+  def getNextLevelAggregatedMetricName(metricName : String, metricRegex: String, metricSuffix: String): String = {
```

Comment:
is metricRegex the delimiter `:::`?

Reply:
yes, it is the delimiter. Let me update the name correctly to reflect it.

Reply:
updated